CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # LoginLdap Changelog
 
+#### LoginLdap 4.7.0
+* Started using SHA-256 instead of MD5 encryption in Base.php
```

Comment:
It would probably be a little more helpful to customers to say something like `while hashing auth secrets` instead of `in Base.php`.

Reply:
@snake14 updated as suggested